contrib/proposal_examples/addcollateral/addcollateral2.json

```diff
@@ -0,0 +1,82 @@
+{
+  "title": "CDP Param Change",
+  "description": "Add collateral XRP",
+  "changes": [
+    {
+      "subspace": "cdp",
+      "key": "CollateralParams",
+      "value": [
+        {
+            "auction_size": "50000000",
+            "conversion_factor": "6",
+            "debt_limit": [
+              {
+                "amount": "1000000000",
+                "denom": "usdx"
+              }
+            ],
+            "denom": "xrp",
+            "liquidation_penalty": "0.05",
+            "liquidation_ratio": "2.0",
+            "market_id": "xrp:usd",
+            "prefix": 0,
+            "stability_fee": "1.000000001547126"
+          },
+          {
+            "auction_size": "10000000",
+            "conversion_factor": "8",
+            "debt_limit": [
+              {
+                "amount": "1000000000000",
+                "denom": "usdx"
+              }
+            ],
+            "denom": "btc",
+            "liquidation_penalty": "0.05",
+            "liquidation_ratio": "1.5",
+            "market_id": "btc:usd",
+            "prefix": 1,
+            "stability_fee": "1.0000000007829977"
+          }
+      ]
+    },
+    {
+      "subspace": "cdp",
+      "key": "DebtParams",
+      "value": [
+          {
+              "conversion_factor": "6",
+              "debt_floor": "10000000",
+              "debt_limit": [
+                {
+                  "amount": "3000000000000",
+                  "denom": "usdx"
+                }
+              ],
+              "denom": "usdx",
+              "reference_asset": "usd"
+          }
+      ]
+    },
+    {
+      "subspace": "cdp",
+      "key": "GlobalDebtLimit",
+      "value": [
+          {
+              "global_debt_limit": [
+                  {
```

Comment:
The value should just be an `sdk.Coins`:

```
"value":  [
                  {
                  "amount": "3000000000000",
                  "denom": "usdx"
                  }
              ]
```

Reply:
https://github.com/Kava-Labs/kava/pull/331/commits/2130feff6fabfb6a510cc26dde0208ac0b8531a0